awx/main/models/jobs.py

```diff
@@ -66,6 +66,13 @@ class JobOptions(BaseModel):
     class Meta:
         abstract = True
 
+    organization = models.ForeignKey(
+        'Organization',
+        blank=True,
+        null=True,
+        on_delete=models.SET_NULL,
+        related_name='%(class)ss',
+    )
```

Comment:
Discussed with @chrismeyersfsu, considering making this a field on the `UnifiedJob` and `UnifiedJobTemplate` models instead. That will involve more migration work, but will be much more performant.